airflow/www/views.py

```diff
@@ -418,12 +418,15 @@ def get_int_arg(value, default=0):
         if request.args.get('reset_tags') is not None:
             flask_session[FILTER_TAGS_COOKIE] = None
             arg_tags_filter = None
+            # Remove the reset_tags=reset from the URL
+            return redirect(url_for('Airflow.index'))
         else:
```

Comment:
Since you're returning in the if block, you can remove the else condition here.

Reply:
Ah, yes, thanks for edits!